### PR TITLE
PI-9384 - fix redirect and images in order history view

### PR DESCRIPTION
**Pull request by awesselburg**
# Pull Request Template

## Description

- redirect to product detail view not working in order history view
- product images not shown in order history view

## Type of change

- [x] Bug fix
- [ ] New feature
- [ ] This change requires a documentation update

## Checklist:

- [ ] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I updated the CHANGELOG.md

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/) and this p
 ## [Unreleased]
 ### Fixed
 - missing taxes on child products during item export
+- fix redirect and images in order history view
```
Comment (Stereck) [inline CHANGELOG.md]: please be more descriptive which redirect and images are fixed here

File: src/modules/shopgate/helpers/shopgate_order_export_helper.php [hunk 2]
```diff
@@ -471,11 +471,22 @@ private function getItems()
     private function getItem(oxOrderArticle $oxOrderArticle, $currency)
     {
         $item = new ShopgateExternalOrderItem();
+        $oxArticle = $oxOrderArticle->getArticle();
+        $isParent = '';
```
Comment (Stereck) [inline src/modules/shopgate/helpers/shopgate_order_export_helper.php]: $isParent suggests to hold a bool value, but in fact it's an itemNumberPrefix

**Review by Stereck — CHANGES_REQUESTED** [review]

File: CHANGELOG.md (hunk 1 above)
Comment (andre-kraus) [reply CHANGELOG.md]: Remove the fix, as it is mention by ### Fixed

File: src/modules/shopgate/helpers/shopgate_order_export_helper.php [hunk 3]
```diff
@@ -471,11 +471,22 @@ private function getItems()
...
+
+        if (
+            !$oxArticle->getParentArticle()
```
Comment (andre-kraus) [inline src/modules/shopgate/helpers/shopgate_order_export_helper.php]: Code style, no need for a line break after (

**Review by andre-kraus — CHANGES_REQUESTED** [review]

**Review by Stereck — APPROVED** [review]

**Review by andre-kraus — APPROVED** [review]

**Pull request merged**